tests/xtro-sharpie/Makefile

```diff
@@ -191,10 +191,10 @@ endif
 gen-all: gen-ios gen-tvos gen-watchos gen-macos
 
 wrench:
-	$(MAKE) -j8 classify
+	$(MAKE) -j8 classify V=1
```

Comment:
Do we still want this bit?

Reply:
I want it to see what is going on, might be too verbose locally